nilearn/reporting/glm_reporter.py

```diff
@@ -173,28 +171,30 @@ def make_glm_report(model,
         Contains the HTML code for the GLM Report.
 
     """
-    '''
+    """
     Bug in Pandas 0.18 : https://github.com/pandas-dev/pandas/issues/13257
     pd.set_option('display.precision', 2)
     limits number of digits shown instead of precision.
     Hence pd.option_context('display.precision', 2) has been used.
```

Comment:
Let's check if this comment is still relevant or needs to be updated because now we support pandas>=1.1.5

Reply:
need to see if there is a test for that and  see what happens if I tweak it.

otherwise I can add a test to prevent any regression